CourseGrab/Home/HomeViewController.swift

```diff
@@ -52,6 +80,10 @@ class HomeViewController: UITableViewController {
         titleLabel.text = "CourseGrab"
         titleLabel.textColor = .white
         titleLabel.font = ._20Semibold
+        let attributes = [
+            NSAttributedString.Key.font: UIFont._20Semibold
+        ]
+        UINavigationBar.appearance().titleTextAttributes = attributes
```

Comment:
If this affects *all* navigation bar appearances, maybe move it to `MainNavigationController`?

Reply:
you can probably also just directly set it without first doing `let attributes = ...` since its only being used once